ansible-role-requirements.yml

```diff
@@ -23,7 +23,19 @@
 - name: rpc-role-logstash
   scm: git
   src: https://github.com/rcbops/rpc-role-logstash.git
-  version: 3f5e7a6044629e76acb1933af19f09a89e55917b
+  version: 5ee5229e9b80e1193b1dd4ba56ef16edf244f237
+- name: rpc-role-filebeat
+  scm: git
+  src: https://github.com/rcbops/rpc-role-filebeat.git
+  version: b2335fa52684c43a42e4e5bfd0bd988d51536457
+- name: rpc-role-kibana
+  scm: git
+  src: https://github.com/rcbops/rpc-role-kibana.git
+  version: 8555c43c67f366e1dfe4beba2844adf69edb65e2
+- name: ansible-elasticsearch
+  scm: git
+  src: https://github.com/d34dh0r53/ansible-elasticsearch
```

Comment:
rpc-o master is ocata, which uses ansible 2.2.x - so the reasoning presented for using the forked role does not apply. Is there another reason we're using the forked role?